lib/config/cli.js

```diff
@@ -24,6 +24,7 @@ function getConfig(argv) {
     boolean: val => val === 'true',
     list: val => val.split(',').map(el => el.trim()),
     string: val => val,
+    integer: parseInt,
```

Comment:
This is a fix for a bug I discovered where integer values weren't being coerced from a string.